plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueToTrinoConverter.java

```diff
@@ -59,6 +62,19 @@
 
     private GlueToTrinoConverter() {}
 
+    public static String getTableType(com.amazonaws.services.glue.model.Table glueTable)
+    {
+        // Athena treats missing table type as EXTERNAL_TABLE.
```

Comment:
is this per Athena doc?

Reply:
I don't remember. This code comment is just moved around.

the change is in response to a NPE because Glue.getTableType happens to be null